xdsl/backend/llvm/__init__.py

```
@@ -0,0 +1,182 @@
+import ctypes
+import subprocess
+
+import llvmlite
```

Let's ignore this error for now
```suggestion
import llvmlite # pyright: ignore[reportMissingTypeStubs]
```

---

Similarly below